src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable max-len */
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonLink } from '../PersonLink/PersonLint';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [error, setError] = useState<boolean>(false);
+  const [loader, setLoader] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState<boolean>(false);
  const [isLoading, setIsLoading] = useState<boolean>(false);
```
for boolean variable names: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig